### ISSUE TITLE
Problem with defining fileAssociations

### ISSUE THREAD

**Issue by Dansoftowner** [petr-panteleyev/jpackage-gradle-plugin#7]
Hi!

Thanks for your great work on this plugin!

I have an issue when defining `fileAssociations`:
```groovy
windows {
        fileAssociations = [ "${projectDir}/distribution/file-associations/file-associations-win.properties" ]
        icon = "${projectDir}/distribution/icon/Boomega.ico"
    }
```

I got:
```
...
Caused by: java.lang.ClassCastException: class org.codehaus.groovy.runtime.GStringImpl cannot be cast to class java.lang.String (org.codehaus.groovy.runtime.GStringImpl is in unnamed module of loader org.gradle.internal.classloader.VisitableURLClassLoader @626b2d4a; java.lang.String is in module java.base of loader 'bootstrap')
	at org.panteleyev.jpackage.JPackageTask.buildParameters(JPackageTask.java:586)
	at org.panteleyev.jpackage.JPackageTask.execute(JPackageTask.java:641)
	at org.panteleyev.jpackage.JPackageTask.action(JPackageTask.java:519)
```

Best regards

**Comment by petr-panteleyev** [comment]
Please check this:
https://issues.gradle.org/browse/GRADLE-2566

**Comment by petr-panteleyev** [comment]
Should be fixed in 1.3.1. I don't use Groovy myself so please check with your build.

**Comment by Dansoftowner** [comment]
Thanks for the great work!
